python/cudnn/gemm/cutedsl/dense/srelu/jax_api.py

```
@@ -0,0 +1,279 @@
+    """Blockscaled GEMM + sReLU as an XLA custom call; usable eagerly or under jax.jit.
+
+    A (M, K, 1) / B (N, K, 1) k-major C-contiguous fp8 arrays, SFA/SFB in the physical
+    atom shape (1, MN', K', 32, 4, 4), prob (M, 1, 1) float32. Returns a plain
+    ``(c_tensor, d_tensor)`` tuple of fresh n-major JAX arrays. fp8 ``d_dtype``
+    configurations (which produce sfd/norm_const outputs) are not reachable from JAX.
+    """
+    c_dtype = _convert_to_cutlass_data_type(c_dtype)
+    d_dtype = _convert_to_cutlass_data_type(d_dtype)
+    acc_dtype = _convert_to_cutlass_data_type(acc_dtype)
+
+    m, _, l = a_tensor.shape
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)